content/docs/manuals/unikernel-operations.md

```diff
@@ -1,12 +1,133 @@
 ---
-title: "Unikernel Operations"
+title: "Unikernel Deployment"
 summary: ""
 draft: false
 weight: 301
 toc: true
 seo:
-  title: "" # custom title (optional)
-  description: "" # custom description (recommended)
+  title: "Unikernel Deployment" # custom title (optional)
+  description: "Deployment of Unikraft Unikernels in Oakestra" # custom description (recommended)
   canonical: "" # custom canonical URL (optional)
   noindex: false # false (default) or true
----
\ No newline at end of file
+---
+
+Oakestra supports the deployment of unikernels built using [Unikraft](http://unikraft.org). 
+
+{{< callout context="caution" title="Requirements" icon="outline/alert-triangle">}}
+
+All you need to start deploying unikernels is:
+
+1. A unikernel .tar.gz hosted somewhere accessible to the Oakestra.
+2. A service deployment descriptor 
+3. A Worker Node with a Unikernel Runtime enabled
+
+{{< /callout >}}
+
+## Packaging your first Unikraft image 
+
+Once you build your first Unikernel using Unikraft, you can create a tarball of the kernel image and the necessary files.
+
+```bash
```

Comment:
```suggestion
```yaml
```

Reply:
that’s actually the output of the `tree` bash command